handbook/community.md

```diff
@@ -188,24 +189,78 @@ graph TD;
     A--D
 ```
 
-## Community contributions (pull requests)
+## Pull Requests
+
+The most important thing when community members contribute to Fleet is to show them we value their time and effort. We need to fget eyes on community pull requests quickly (within 1 business day) and get them pushed merged as soon as we can.
+
+
+
+### Process for managing community contributions
+
+The Community Engagememt DRI is responsible for keeping an eye out for new community contributions and either getting them merged in if possible and get the right eyes on them if they need to be reviewed. 
+
+Each business day, the Community Engagement DRI will check open pull requests to:
+
+1. Check for new pull requests (PRs) from the Fleet community. 
+2. Approve and merge any community PRs that are ready to go.
+3. Make sure there aren't any existing community PRs waiting for followup from Fleet. 
+
+#### Identify community contributions
+
+When a new pull request is submitted by a community contributor (someone not a member of the Fleet organization):
+
+- Add the `:community` label.
+- Self-assign for review.
+- Check whether the PR can be merged, or needs to be reviewed by the Product team.
+    - Things that generally don't need additional review:
+        - Minor changes to the docs.
+        - Small bug fixes.
+        - Additions or fixes to the Standard Query Library(SQL) are (as long as the SQL works properly and they are attributed correctly).
+        - Use the Please refer to our [PRs from the community](https://docs.google.com/document/d/13r0vEhs9LOBdxWQWdZ8n5Ff9cyB3hQkTjI5OhcrHjVo/edit?usp=sharing) guide and use your best judgement. 
+    - If a review is needed:
+        - [Product DRIs](./product.md#product-dris) should approve extensive changes and new features. Ask in the #g-product channel in Fleet's Slack for more information.
+        - Tag the DRI and the contributor in a comment on the PR letting everyone know why additional review is needed. Make sure to say thanks!
```

Comment:
Also use the "request review" feature for the DRI.

Reply:
Done!